packages/react-components/src/templates/SharedResearchOutput.tsx

```diff
@@ -39,7 +39,7 @@ const SharedResearchOutput: React.FC<SharedResearchOutputProps> = ({
   ...props
 }) => {
   const isGrantDocument = (
-    ['Grant Document', 'Proposal'] as ResearchOutputType[]
+    ['Grant Document', 'Proposal', 'Presentation'] as ResearchOutputType[]
```

Comment:
this needs to be changed after merge https://github.com/yldio/asap-hub/pull/1194